### PR TITLE
add ens domains for smart-contracts on bootnode file

### PR DESCRIPTION
**Pull request by jordipainan**
docs -> https://docs.vocdoni.io/#/architecture/services/bootnode

### CODE REVIEW

**Comment by brickpop** [comment]
As commented in person, the PR itself looks good, but in practice it is not being used now.
Web clients don't use it and mobile apps are not in use right now

File: src/client-store/gateway.proto [hunk 1]
```diff
@@ -13,9 +13,21 @@ message BootNodeGateways {
             string pubKey = 3;
         }
         message Web3 {
-            string uri = 1;
+            message Endpoints {
+                string uri = 1;
+            }
+            message Contracts {
+                string publicRegistry = 1;
```
Comment (brickpop) [inline src/client-store/gateway.proto:20]: I would call them something like:
- `ensRegistryAddress`
- `ensPublicResolverAddress`
- `entityResolver`
- (etc)

Comment (jordipainan) [reply src/client-store/gateway.proto]: done :)

**Comment by jordipainan** [comment]
closing due to changes on the protocol

**Pull request closed without merging**